man/docker-run.1.md

```diff
@@ -100,6 +101,9 @@ option can be set multiple times.
 **--blkio-weight**=0
    Block IO weight (relative weight) accepts a weight value between 10 and 1000.
 
+**--blkio-weight-device**=[]
+   Block IO weight (relative weight for per device).
```

Comment:
can you add (device:weight)?

Perhaps the man page should also contain an example?